lib/temporal/activity/context.rb

```diff
@@ -56,6 +56,10 @@ def headers
         metadata.headers
       end
 
+      def log_tag
+        "#{metadata.namespace} / #{metadata.workflow_name}[#{metadata.workflow_id}:#{metadata.workflow_run_id}] / #{metadata.name}[#{metadata.id}]##{metadata.attempt}"
```

Comment:
this can be generated once, no need to generate it every time the method is called

Reply:
Method removed.